kombu/transport/redis.py

```diff
@@ -1102,8 +1102,9 @@ def _sentinel_managed_pool(self, asynchronous=False):
         additional_params.pop('port', None)
         connection_list = []
         for url in self.connection.client.alt:
-            if url and 'sentinel://' in url:
```

Comment:
I must admit I'm a bit unsure why the `if url` check was in there.

Reply:
Perhaps there were `None` results in the list beforehand? Not sure why either.

Reply:
Ping @badcure as the author of the check.

Reply:
This `for` loop is mentioned in https://github.com/celery/kombu/issues/1004 as well.

Reply:
So I figured this out myself. Apparently `url` can be an empty string. This happens if you provade an URL string like this: `"sentinel://localhost:65532/;"` (i.e. a trailing semi-colon).

But it's not a problem becuase `url.scheme` will also be an empty string:

```python
>>> from kombu.utils.url import _parse_url
>>> _parse_url('')
urlparts(scheme='', hostname=None, port=None, username=None, password=None, path=None, query={})
```